.github/helpers/jira-api/refine_tickets.py

```diff
@@ -0,0 +1,221 @@
+""" Importing modules for parsing, and json formatting"""
+import re
+import sys
+import json
+import datetime
+
+JIRA_SUBTASK = "10003"
+
+###################################################################################################
+##
+## This script hosts functions that are used to reformat given strings, create specific format and
+## post parent and subtask tickets.
+##
+###################################################################################################
+
+def break_update_down( update ):
+    """
+    Takes in a string with an update detailed and rearanges to the format we want
+      "Update <dependency> from `<old version>` to `<new version>'
+    
+    Args:
+      update (string): string holding current dependency update
+
+    Returns:
+      summary (string): reformated string holding current dependency update
+    """
+
+    # seperate dependency into 2 groups
+    check_str = re.search( r"^- `(.*)` Update (from version `.*` to `.*`)", update )
+    # reformat line
+    summary = "Update " + check_str.group(1) + " " + check_str.group(2)
+    return summary
+
+
+def create_parent_ticket( project_key, updates ):
+    """
+    POST API to create a parent ticket on the specified board
+
+    Args:
+      conn (HTTPSConnection): specifies where to make the connection
+      headers (string): supplies authentication to connect to JIRA
+      project_key (string): defines project key of JIRA board we want to post to
+      updates (tuple(lists)): tuple of the three update lists
```

Comment:
Looks like these args could be removed:
```suggestion
      project_key (string): defines project key of JIRA board we want to post to
      updates (tuple(lists)): tuple of the three update lists
```